<launch>
    <node name="lsd_node_dataset" pkg="lsd_slam_core" type="dataset_slam" respawn="false" output="screen">

	    <param name="hz" value="0" />
	    <param name="calib" value="/misc/lmbraid19/thomasa/rosbuild_ws/packages/lsd_slam/lsd_slam_core/calib/OpenCV_example_calib.cfg" />
	    <param name="files" value="/misc/lmbraid19/thomasa/datasets/rgbd/rgbd_dataset_freiburg1_teddy/rgb_depth_associated.txt"/>

	    <param name="doSLAM" value="true"/>
	    <param name="/Debug/plotStereoImages" value="false"/>
	    <param name="/Debug/printTrackingIterationInfo" value="false"/>
    
    </node>
    
    <node name="lsd_viewer" pkg="lsd_slam_viewer" type="viewer" respawn="true" output="screen"/>

    <node name="siasa_viewer" pkg="reinforced_visual_slam" type="siasa_viewer.py" respawn="true" output="screen">
        <param name="gt" value="/misc/lmbraid19/thomasa/datasets/rgbd/rgbd_dataset_freiburg1_teddy/groundtruth_associated.txt"/>
        <remap from="lsd_slam/keyframes" to="/lsd_slam_reinforced/keyframes"/>
    </node>

</launch>
